benchmark_entry.py

```diff
@@ -0,0 +1,9 @@
+from benchopt.benchmark import Benchmark
```

Comment:
all of these files make the diff harder to parse and the PR harder to review. You can have a separate benchmark PR in your fork of benchopt_logreg_l1